services/workflows-service/src/workflow/workflow.service.ts

```
@@ -1596,6 +1596,24 @@ export class WorkflowService {
             },
           });
 
+          const collectionFlow = buildCollectionFlowState({
```

Biome (1.9.4)

[error] Shouldn't redeclare 'collectionFlow'. Consider to delete it or rename it.

'collectionFlow' is defined here:


(lint/suspicious/noRedeclare)